.github/workflows/pyoxidizer.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: Build Linux binary using PyOxidizer
+
+on:
+  push:
+    tags: ["*"]
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Linux binary
+    # For our binary to work on esbench (which defaults to Ubuntu 18.04), we
+    # need to link our executable with the glibc that ships with Ubuntu 18.04.
+    # The easiest way to do this is to actually build the binary from inside
+    # Ubuntu 18.04. Since GitHub Actions is removing Ubuntu 18.04, we build our
+    # executable in an Ubuntu 18.04 Docker container instead.
```

Comment:
> # For our binary to work on Ubuntu 18.04, we need to link our executable

It's important to note that it will also work with later versions, such as Ubuntu 20.04 and 22.04, and other distributions. Maybe I should use "For our binary to work on Ubuntu 18.04 and above"?